contracts/Rln.sol

```diff
@@ -86,7 +88,8 @@ contract RLN {
 		members[_pubkeyIndex] = 0;
 
 		// refund deposit
-		(bool sent, _) = receiver.call{value: MEMBERSHIP_DEPOSIT}("");
+		uint256 withdrawableDeposit = MEMBERSHIP_DEPOSIT * (WAD - PROTOCOL_FEES_PERCENTAGE) / WAD
+		(bool sent, _) = receiver.call{value: withdrawableDeposit}("");
```

Comment:
Please also add comments next to each code line to elaborate on what is happening.

Reply:
> Please also add comments next to each code line to elaborate on what is happening.

@kgrgpg  Once the comment is addressed, let me know to re-review.